generate.jl

```diff
@@ -0,0 +1,27 @@
+using Pkg
+Pkg.activate(".")
+using Fire
+using PhysicsTutorials
+
+path_names(path::String) = path in ("","/") ? () : (path_names(dirname(path))..., basename(path))
```

Comment:
Use [`splitpath`](https://docs.julialang.org/en/latest/base/file/#Base.Filesystem.splitpath).

Reply:
However, note that it has only been added in v1.1. So perhaps define a custom `splitpath` in a `if VERSION <= v"1.1"` block.

Also, note that your `path_names` function doesn't work on Windows, i.e. try `path_names("C:\\test")`.

Reply:
I was intended to use `path_names` to solve the compatibility issue, but didn't test for windows users. Now I copy paste the Julia 1.1 implementation of `splitpath` to `convert.jl`, this works pretty well for both Julia 1.0 and julia 1.1 and hopefully work for Windows users too.